ntoskrnl/ke/bug.c

```diff
@@ -1080,14 +1080,7 @@ KeBugCheckWithTf(IN ULONG BugCheckCode,
         KeBugCheckOwner = Prcb->Number;
 
         /* Freeze the other CPUs */
-        for (ULONG i = 0; i < KeNumberProcessors; i++)
-        {
-            if (i != Prcb->Number)
-            {
-                /* Send the IPI */
-                KiIpiSend(AFFINITY_MASK(i), IPI_FREEZE);
-            }
-        }
+        KxFreezeExecution();
 
         /* Give the other CPUs one second to catch up */
         KeStallExecutionProcessor(1000000);
```

Comment:
I guess you could remove that if you call KxFreezeExecution() now, since that function actually checks explicitly that all the CPUs (except the caller) become frozen.